src/app/frontend/common/components/labels/labels_controller.js

```diff
@@ -19,6 +19,12 @@
 const alwaysVisibleLabelsNumber = 5;
 
 /**
+ * Regular expression matching URLs.
+ * @type {RegExp}
+ */
+const urlRegexp = new RegExp('(https?|ftp)://(-\.)?([^\s/?\.#-]+\.?)+(/[^\s]*)?', 'i');
```

Comment:
Plus the fact, that at the moment only value is clickable link, not whole chip with annotation.